.github/workflows/Lockfile.yml

```diff
@@ -1,6 +1,9 @@
 name: Lockfile
 on:
   pull_request:
```

Comment:
I think we should remove "pull_request" here

Reply:
Should we separate it into two workflows, one that commits a new lockfile on push to main, and one that verifies that the PR has a correct lockfile (and could commit a new one to the source branch if the PR is this repo).

We then technically do it twice and the one that runs on pushes to main would be a safety if something would get merged without an updated lockfile.

Or skip validation of the lockfile on PRs and simply update it on every push to main?

Reply:
> Should we separate it into two workflows

yes, completely, that would be much cleaner. and the modification one would be removed once renovate will be able to update both the pom and the lockfile at the same time.